ml-algorithms/src/main/java/org/opensearch/ml/engine/ModelHelper.java

```diff
@@ -135,6 +135,12 @@ public void downloadPrebuiltModelConfig(
                                     case TextEmbeddingModelConfig.MODEL_MAX_LENGTH_FIELD:
                                         configBuilder.modelMaxLength(((Double) configEntry.getValue()).intValue());
                                         break;
+                                    case TextEmbeddingModelConfig.QUERY_PREFIX:
+                                        configBuilder.queryPrefix(configEntry.getValue().toString());
+                                        break;
```

Comment:
add test.

Reply:
This one is a chicken-and-egg problem. This is only testable when an asymmetric model is added to the OpenSearch model server, from which one can download prepackaged models. I can remove this code, but then it will not be possible to use an uploaded model, if the community decides to prepackage one. 

Shall I remove this code?

Reply:
Leave this for now then. Thanks for the explanation.